[1,  1000] loss: 0.240
Traceback (most recent call last):
  File "D:\Backup\Less Important\My programs\Git\ML\Models\CNN\dogs_vs_cats\main.py", line 76, in <module>
    train_model(
  File "D:\Backup\Less Important\My programs\Git\ML\Models\CNN\dogs_vs_cats\dvc_train\train.py", line 50, in train_model
    outputs = cnn(inputs)
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "D:\Backup\Less Important\My programs\Git\ML\Models\CNN\dogs_vs_cats\dvc_model\cnn.py", line 25, in forward
    x = self.pool(F.relu(self.conv2(x)))
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\conv.py", line 554, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "C:\Users\vav11\anaconda3\envs\Conda310\lib\site-packages\torch\nn\modules\conv.py", line 549, in _conv_forward
    return F.conv2d(
KeyboardInterrupt
